src/prop/prop_proof_manager.cpp

```diff
@@ -73,7 +109,7 @@ void PropPfManager::checkProof(const context::CDList<Node>& assertions)
 std::vector<Node> PropPfManager::getUnsatCoreLemmas()
 {
   std::vector<Node> usedLemmas;
-  std::vector<Node> allLemmas = d_proofCnfStream->getLemmaClauses();
+  std::vector<Node> allLemmas = getLemmaClauses();
```

Comment:
since the lemmas are part of this clause now, there should be no need to convert into vector before iterating in the loop below.

Reply:
This is necessary since context::CDHashSet does not have the appropriate iterators.